### PR TITLE
Feat: visual component for Device List Toolbar

### PR DESCRIPTION
**Pull request by rkpattnaik780**
### Type of change

- Visual component

### Description

The visual component for Device List Toolbar along with its storybook.

### Screenshot

Screenshot from 2020-05-11 17-21-04

### CODE REVIEW

File: console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          />
+        </DataToolbarItem>
+        <DataToolbarItem>
+          <CreateDeviceButton
+            handleInputDeviceInfo={handleInputDeviceInfo}
+            handleJSONUpload={handleJSONUpload}
+          />
+        </DataToolbarItem>
+        <DataToolbarItem>
+          <DropdownWithKebabToggle isPlain={true} dropdownItems={kebabItems} />
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx:68]: key and ariaLabel are missing.

File: console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        </DataToolbarItem>
+        <DataToolbarItem
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx:70]: id is missing

File: console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  handleOnToggle,
+  isChecked,
+  items,
+  handleOnChange,
+  kebabItems
+}) => {
+  return (
+    <DataToolbar id="device-data-toolbar">
+      <DataToolbarContent id="device-data-toolbar-content">
+        <DataToolbarItem variant="bulk-select">
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/modules/device/components/DeviceListToolbar/DeviceListToolbar.tsx:50]: id is missing

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx [hunk 4]
```diff
@@ -24,7 +23,6 @@ export const BulkSelect: React.FunctionComponent<IBulkSelectProps> = ({
   isChecked,
```
Comment (ajaypratap003) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:23]: not sure about the purpose this component but i can see that you are using PF Dropdown component. can we try component name something else like  DropdownWithBulkSelect.

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx [hunk 5]
```diff
@@ -43,9 +41,7 @@ export const BulkSelect: React.FunctionComponent<IBulkSelectProps> = ({
               aria-label="Select all items"
```
Comment (ajaypratap003) [inline console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:41]: id's and aria-label should be get as props, not hard code.

File: console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx (hunk 4 above)
Comment (rkpattnaik780) [reply console/console-init/ui/src/components/BulkSelect/BulkSelect.tsx:23]: This is for [BulkSelect](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/bulk-selection#integrated-bulk-selection-for-tables).

**Pull request merged**